terminus-ui/tabs/testing/src/test-components.ts

```diff
@@ -179,7 +179,7 @@ export class AsyncTabs implements OnInit {
 export class SimpleLabels {
   public otherLabel = 'Bar';
   public otherContent = 'bar content';
-  @ViewChild('test') public testSelector: any;
+  @ViewChild('test', {static: true}) public testSelector: any;
```

Comment:
is this public declaration supposed to move down? is that the rule or just preference?

Reply:
There's no lint rule for this, (we should write one) but it has been enforced in component files by review - just not in test files. I am open though if we'd prefer the same format rules in test components too